src/components/SceneStats/Stats/Stats.types.ts

```diff
@@ -1,3 +1,4 @@
 export type Props = {
   label: string
+  children?: React.ReactNode
```

Comment:
wdyt about using `export type Props = React.PropsWithChildren` in this component?

Reply:
IMO both valid but React.ReactNode more explicit